dev/test/bulk-writer.ts

```diff
@@ -348,23 +332,20 @@ describe('BulkWriter', () => {
     expect(() => bulkWriter.close()).to.throw(expected);
   });
 
-  it('sends writes to the same document in separate batches', async () => {
+  it('sends writes to the same documents in the same batch', async () => {
```

Comment:
Does this test still have value over the other tests?

Reply:
This test would fail if pendingOps got changed back to its original map type since the second write to the same document path would override the existing entry, so it serves to make sure that writes to the same document are added properly.

Reply:
Makes sense. Maybe prefix the test name with "can send"?

Reply:
done.